packages/@webex/plugin-meetings/package.json

```diff
@@ -12,7 +12,7 @@
     "Taymoor Khan <taykhan@cisco.com>"
   ],
   "main": "dist/index.js",
-  "types": "types/index.d.ts",
```

Comment:
so both the types and code are in the same folder we dont need seperate types folder !?